### PR TITLE
Change crossgen defaults

### PR DESCRIPTION
**Pull request by jkotas**
Change crossgen to generate R2R images by default

### CODE REVIEW

**Comment by jkotas** [comment]
@gkhanna79 PTLA

**Comment by davidfowl** [comment]
Whyyy

File: src/tools/crossgen/crossgen.cpp [hunk 1]
```diff
@@ -591,6 +596,10 @@ int _cdecl wmain(int argc, __in_ecount(argc) WCHAR **argv)
         {
             dwFlags |= NGENWORKER_FLAGS_READYTORUN;
         }
+        else if (MatchParameter(*argv, W("FragileNonVersionable")))
```
Comment (JohnChen0) [inline src/tools/crossgen/crossgen.cpp]: Should we add this option to the usage output, or are we intentionally making it hard to discover?

**Comment by JohnChen0** [comment]
LGTM

File: src/tools/crossgen/crossgen.cpp (hunk 1 above)
Comment (jkotas) [inline src/tools/crossgen/crossgen.cpp]: I would keep it out - it won't be the only hidden switch.

**Comment by jkotas** [comment]
> Whyyy

Because of that's what we want to converge to.

**Comment by gkhanna79** [comment]
LGTM

**Pull request merged**